packages/fluent-editor/src/modules/ai/index.ts

```
@@ -32,125 +122,537 @@
+    this.quill.container.appendChild(this.dialogContainerEl)
+  }
+
+  // 创建结果弹窗
+  private createResultElement() {
+    if (!this.resultPopupEl) {
+      this.resultPopupEl = document.createElement('div')
+      this.resultPopupEl.className = 'ql-ai-result'
+      this.resultPopupEl.style.display = 'none'
+      this.resultPopupHeaderEl = document.createElement('div')
+      this.resultPopupHeaderEl.className = 'ql-ai-result-header'
+      this.resultPopupHeaderEl.innerText = RESULT_HEADER_TEXT
```

ESLint

[error] Prefer `.textContent` over `.innerText`.

(unicorn/prefer-dom-node-text-content)